examples/gcp-metrics-integration/main.tf

```diff
@@ -0,0 +1,7 @@
+resource "chronosphere_gcp_metrics_integration" "customer" {
+  name = "GCP Metrics Integration"
+  slug = "test-gcp-metrics-integration"
```

Comment:
for follow-up, this example is pretty light, doesn't have any metric_groups which seems to be the main entity body
cc @kirgirchrono